### PR TITLE
python3Packages.ripe-atlas-cousteau: init at 1.5.1

### PR DESCRIPTION
**Pull request by RaitoBezarius**
Depends on #155269

###### Motivation for this change

This adds the client library for RIPE ATLAS API, which is necessary for `ripe-atlas-tools`, see #155267 for context.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/ripe-atlas-cousteau/default.nix [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+, fetchFromGitHub
+}:
+buildPythonPackage rec {
+  pname = "ripe-atlas-cousteau";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "RIPE-NCC";
+    repo = pname;
+    # TODO: remove this if when https://github.com/RIPE-NCC/ripe-atlas-cousteau/issues/59 is closed.
```
Comment (applePrincess) [inline pkgs/development/python-modules/ripe-atlas-cousteau/default.nix:22]: The issue referencing is closed.

Comment (RaitoBezarius) [reply pkgs/development/python-modules/ripe-atlas-cousteau/default.nix]: This is fixed.

File: pkgs/development/python-modules/ripe-atlas-cousteau/default.nix [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    pytestCheckHook
+    jsonschema
+  ];
+
+  pythonImportsCheck = [
+    "ripe.atlas.cousteau"
+  ];
+
+  meta = with lib; {
+    description = "A Python client library for RIPE ATLAS API";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/ripe-atlas-cousteau/default.nix:44]: Please strip A python.

File: pkgs/development/python-modules/ripe-atlas-cousteau/default.nix [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+, python-dateutil
+, python-socketio
+, requests
+, jsonschema
+, pythonOlder
+, pytestCheckHook
+, buildPythonPackage
+, fetchFromGitHub
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/ripe-atlas-cousteau/default.nix:11]:
```suggestion
}:

buildPythonPackage rec {
```

**Comment by SuperSandro2000** [comment]
@ofborg build python310Packages.ripe-atlas-cousteau

**Pull request merged**